### PR TITLE
Applied lesson layout

### PR DESCRIPTION
**Pull request by apawlik**
Applied the lesson template to make it look like http://apawlik.github.io/R-ecology/

Some more cleaning up and lesson linking needed but at least the index page gets the logo and layout (roughly OK).

### CODE REVIEW

**Comment by tracykteal** [comment]
The list of lessons still includes the Lesson 00 Name and Lesson 01 Name. I also think we want this list to be numeric.

Also, do we want to call this lesson 'Starting with R for data analysis for Ecology' or just 'R for Ecology'?

**Comment by apawlik** [comment]
Sorry to mix in two things - the ggplot2 lesson adding and lesson styling. But at least the lessons are now listed correctly in index. I'm having problems generating html from Rmd for ggplot2 lesson. Don't know knitr :-(

**Comment by apawlik** [comment]
OK. I managed to produce html for ggplot2 - PR landing soon

File: css/swc.css [hunk 1]
```diff
@@ -73,7 +73,11 @@ body.stylesheet {
 div.banner {
     background-color: #FFFFFF;
     width: 100%;
+<<<<<<< HEAD
     height: 90px;
```
Comment (fmichonneau) [inline css/swc.css]: There is a conflict here

File: index.md [hunk 2]
```diff
@@ -11,43 +26,58 @@ import CSV files, the structure of data.frame, how to deal with factors, how to
...
+{% else %}
+{% include anySetup.html %}
+{% endif %}
 
-The lessons are written in Rmarkdown. A Makefile generates an html page for each
-topic using knitr. In the process, knitr creates an intermediate markdown
-file. These are removed by the Makefile to avoid clutter.
+<p><strong>Twitter</strong>: @datacarpentry</p>
 
-The Makefile also generates a "skeleton" file that is intended to be distributed
```
Comment (fmichonneau) [inline index.md]: If we remove this from here, we should make sure this info is included somewhere else.

Comment (apawlik) [inline index.md]: Maybe in README?

File: css/swc.css (hunk 1 above)
Comment (apawlik) [inline css/swc.css]: Ugh, sorry - though I checkout out the cleaned version. Let me fix this.

File: index.md (hunk 2 above)
Comment (fmichonneau) [inline index.md]: Yes, that works. The Makefile would need to updated though, as currently README and index are kept in sync by it.

**Comment by tracykteal** [comment]
I was just going to do the formatting and saw that this PR was already here and had been sitting for awhile. I'm just going to merge it as most of it looks good.

**Pull request merged**